packages/mui-types/index.d.ts

```diff
@@ -146,3 +146,10 @@ export interface OverridableTypeMap {
  */
 // tslint:disable-next-line: ban-types
 export type Simplify<T> = T extends Function ? T : { [K in keyof T]: T[K] };
+
+/**
+ * Changes the properties K from T to required
+ */
+export type PickRequired<T, K extends keyof T> = DistributiveOmit<T, K> & {
```

Comment:
The implementation is OK, but the name is confusing - "Pick" suggests it discards other properties.
`PartiallyRequired` comes to my mind as a better alternative, but if anyone has a better option, feel free to suggest.

Reply:
@michaldudak `PartiallyRequired` also sounds better to me. I will fix this.